test/end2end/model_test.dart

```diff
@@ -628,11 +628,22 @@ void main() {
           fakeLibrary.enums.firstWhere((e) => e.name == 'MacrosFromAccessors');
       macroReferencedHere = MacrosFromAccessors.publicConstantFields
           .firstWhere((e) => e.name == 'macroReferencedHere');
+      ClassTemplateOneLiner = exLibrary.allClasses
+          .firstWhere((c) => c.name == 'ClassTemplateOneLiner');
+    });
+
+    test('via reexport does not leave behind template crumbs', () {
+      expect(ClassTemplateOneLiner.isCanonical, isFalse);
+      expect(
+          ClassTemplateOneLiner.oneLineDoc,
+          equals(
+              'I had better not have a template directive in my one liner.'));
     });
 
     test('renders a macro defined within a enum', () {
       expect(macroReferencedHere.documentationAsHtml,
           contains('This is a macro defined in an Enum accessor.'));
+      [1, 2, 3].whereType();
```

Comment:
Eh?

Reply:
what in the.... I have no idea what this is from.

Reply:
sometimes i think vs code is randomly inserting things on me, in part because I haven't yet managed to get the control scheme to be truly vim like.